doc/nrf/libraries/bluetooth_services/mesh/sensor.rst

```diff
@@ -214,6 +214,26 @@ A sensor's Descriptor contains parameters that may aid other mesh nodes in inter
 
 The sensor descriptor is constant throughout the sensor's lifetime.
 If the sensor has a descriptor, a pointer to it should be passed to :c:member:`bt_mesh_sensor.descriptor` on init.
+See the following small sample:
+
+.. code-block:: c
+
+   static const struct bt_mesh_sensor_descriptor temp_sensor_descriptor = {
+       .tolerance = {
+           .negative = { .val1 = 0, .val2 = 750000 },
+           .positive = { .val1 = 3, .val2 = 500000 },
+       },
+       .sampling_type = BT_MESH_SENSOR_SAMPLING_ARITHMETIC_MEAN,
+       .period = 300,
+       .update_interval = 50
+   };
+
+   struct bt_mesh_sensor temp_sensor = {
+       .type = &bt_mesh_sensor_present_dev_op_temp,
```

Comment:
We have a description of the descriptor, but not type. Should we add "bt_mesh_sensor_present_dev_op_temp" too? Or is it to much information, as we maybe have to do this for all 3 places where type is used.

Reply:
The type(s) are described in the sensor_types.rst document. What do you suggest to add here?